.travis.yml

```diff
@@ -8,11 +8,4 @@ rvm:
   - 2.3.1
   - 2.2.6
   - 2.1
-  - 2.0
```

Comment:
Pls consider updating the [Readme](https://github.com/mkdynamic/omniauth-facebook/blob/master/README.md#supported-rubies) too. `MRI Ruby 1.9.3` support must be dropped as `omniauth-oauth2` v1.5.0 started to use the `%i[]` notation which was introduced only in Ruby 2.0.0, I'm not sure about the others, are those necessary too (in this scope)?